### PR TITLE
Add Year Range Picker

### PR DESCRIPTION
**Pull request by clariokids**
Please check @mathieustan 
- Adding Year picker to be more dynamic, as default is 10 and must be number
- Adding documentation and sample "Year Range Picker"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 ## Demo
 
-To view a demo online: <https://vue-datepicker.netlify.com/>
```
Comment (mathieustan) [inline README.md]: hmmm you removed alot in README. 

Rollback this file.

File: README.md [hunk 2]
```diff
@@ -113,6 +100,7 @@ Using `v-model`
 | fullscreenMobile | Boolean              | false            | Datepicker will appear in a bottom sheet in mobile view                                                                                                                                                                 |
 | minDate          | String\|Number\|Date |                  | Date are available FROM this date                                                                                                                                                                                       |
 | maxDate          | String\|Number\|Date |                  | Date are disabled AFTER this date                                                                                                                                                                                       |
+| yearRangePicker  | Number               | 10               | Year picker range                                                                                                                                                                                                       |
```
Comment (mathieustan) [inline README.md]: visibleYears or visibleYearsNumber would be more appropriate.

File: src/components/datepicker/DatePickerYearMonth.vue [hunk 3]
```diff
@@ -126,14 +126,15 @@ export default {
...
+    yearRangePicker: { type: Number, default: 10 },
   },
   computed: {
     yearFormatted () {
       return this.currentDate.getYearFormatted();
     },
     getYears () {
       const year = this.currentDate.year;
-      return this.currentDate.generateYearsRange(year, 10, 'year');
+      return this.currentDate.generateYearsRange(year, this.yearRangePicker, 'year');
```
Comment (mathieustan) [inline src/components/datepicker/DatePickerYearMonth.vue]: Can you add a test for this computed.

File: README.md (hunk 1 above)
Comment (clariokids) [reply README.md]: ok, it's just because i'm using some "markdown beautifier" when trying to add about this new prop

File: README.md (hunk 2 above)
Comment (clariokids) [reply README.md]: yes, bit struggling in choosing appropriate name >.<

File: src/components/datepicker/DatePickerYearMonth.vue (hunk 3 above)
Comment (clariokids) [reply src/components/datepicker/DatePickerYearMonth.vue]: ok

**Comment by clariokids** [comment]
@mathieustan please check again, update some your request

**Comment by mathieustan** [comment]
Closed but added in 0.2.0. Thanks alot for your time, and sorry for delay.

**Pull request closed without merging**